datasets/gg/disqualified_directors/crawler.py

```diff
@@ -30,13 +33,91 @@ def parse_table(table: _Element) -> Generator[Dict[str, str], None, None]:
         yield {hdr: c for hdr, c in zip(headers, cells)}
 
 
+def parse_html(doc: etree.ElementTree):
+    items = doc.findall('.//details[@class="helix-item helix-item-accordion"]')
+    if not items:
+        raise ValueError("Cannot find any details")
+
+    for item in items:
+        # Extract the summary and content details
+        summary = item.find('.//summary/h3[@class="item-title"]')
+        content = item.find('.//div[@class="generic-content field--name-copy"]')
+
+        # Extract name, DOB, and address
+        name_info = summary.text.strip()
+        title_match = re.search(
+            r"^(?P<name>.*?)\s*\(?\s*[Dd]ate of Birth\s*(?P<dob>\d{1,2}\s+[A-Za-z]+\s+\d{4})\s*\)?\s+of\s+(?P<address>.*)$",
+            name_info,
+        )
+        if not title_match:
+            continue
+
+        name = title_match.group(1).strip()
+        birth_date = title_match.group(2).strip()
+        address = title_match.group(3).strip()
```

Comment:
it might be nice to also add something like

```python
if "Guernsey" in address:
    person.add("country", "gg")
```

I wouldn't try and parse the country out of the address, we have broader address parsing ideas and that's not something to solve with the irregular addresses here. But that little `if` covers most of these people.